### PR TITLE
Add Vivaldi v1.0.83.38 cask

### PR DESCRIPTION
**Pull request by zchee**

### CODE REVIEW

File: Casks/vivaldi.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask :v1 => 'vivaldi' do
+  version '1.0.83.38'
+  sha256 'ef048f271526c53574f453dfdcac157482fc51e9f669c22aadaf1b25b5f952fa'
+
+  url "https://vivaldi.com/download/Vivaldi_TP_#{version}.dmg"
+  homepage 'http://www.play0ad.com/'
```
Comment (vitorgalvao) [inline Casks/vivaldi.rb]: Homepage is wrong.

File: Casks/vivaldi.rb [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+  license :unknown
```
Comment (vitorgalvao) [inline Casks/vivaldi.rb]: Could you please add a license? `:gratis`, I think.

**Comment by ndr-qef** [comment]
Sorry, this happens to be a duplicate of #9279.

**Comment by zchee** [comment]
ok :)
thanks

**Pull request closed without merging**